test/plots/cars-mpg.js

```diff
@@ -0,0 +1,35 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const data = await d3.csv("data/cars.csv", d3.autoType);
+
+  return Plot.plot({
+    x: {inset: 20, ticks: 10},
+    y: {grid: true},
+    color: {
+      type: "categorical",
+      legend: true // as soon as it's available
+    },
+    marks: [
+      Plot.ruleY([0], {filter: false}),
+      Plot.line(data,
+        Plot.binX({y: "mean"}, {
+          x: d => +d["year"],
```

Comment:
Is the coercion and function accessor necessary here?

```suggestion
          x: "year",
```